### ISSUE TITLE
ModSpec: metadata list should respect capitalization of custom attributes

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/metanorma-iso#805]
Code:

```adoc
[conformance_class]
.{{item.name}} ({{group.standard}})
====
[%metadata]
identifier:: {{item.identifier}}
description:: {{item.scope}}
Namespace prefix:: {{item.abbreviation}}
Namespace URI:: {{item.identifier}}
UML package source:: {{item.source}}
UML package name:: {{item.uml-package}}
====
```

<img width="767" alt="Screen Shot 2022-09-29 at 3 20 22 PM" src="https://user-images.githubusercontent.com/11865/192966016-654ad020-e5e5-459c-940e-8cb0ee7fc138.png">

Notice how "URI" => "Uri" and "UML" => "Uml".

**Comment by ronaldtse** [comment]
@opoudjis I'm running Git gems but still seeing this issue. How should I fix this?

**Comment by ronaldtse** [comment]
Actually this is fixed in #813 . Thanks!
